### PR TITLE
Added missing validations in add tenant modal

### PR DESCRIPTION
**Pull request by bexsoft**
## What does this do?

Added missing validations in add tenant modal

## How to test?

- Open add tenant modal
- Verify that required fields ar correctly validated

### CODE REVIEW

File: portal-ui/src/screens/Console/Tenants/ListTenants/ListTenants.tsx [hunk 1]
```diff
@@ -238,7 +236,7 @@ const ListTenants = ({ classes }: ITenantsList) => {
         />
       )}
       {showNewCredentials && (
-        <TenantCredentialsPrompt
+        <CredentialsPrompt
```
Comment (dvaldivia) [inline portal-ui/src/screens/Console/Tenants/ListTenants/ListTenants.tsx:239]: I don't see changes on the `CredentialsPrompt` component, will it also display the console credentials if they are available?

**Review by dvaldivia — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Tenants/ListTenants/ListTenants.tsx (hunk 1 above)
Comment (bexsoft) [reply portal-ui/src/screens/Console/Tenants/ListTenants/ListTenants.tsx:239]: Yes, Credentials prompt will show them, also TenantCredentialsPrompt is missing on master, so this change is required.

**Review by dvaldivia — APPROVED** [review]

File: portal-ui/src/screens/Console/Tenants/ListTenants/ListTenants.tsx (hunk 1 above)
Comment (bexsoft) [reply portal-ui/src/screens/Console/Tenants/ListTenants/ListTenants.tsx:239]: Fixed

**Review by dvaldivia — APPROVED** [review]

**Review by Alevsk — DISMISSED** [review]

**Comment by bexsoft** [comment]
Changes merged in #289

**Pull request closed without merging**